app/controllers/concerns/vet360/writeable.rb

```diff
@@ -48,6 +48,10 @@ def build_record(type, params)
       # This needs to be refactored after V2 upgrade is complete
       if type == 'address' && Flipper.enabled?(:remove_pciu, @current_user)
         model = 'VAProfile::Models::V3::Address'
+
+        # Validation Key was deprecated with ContactInformationV2
+        params[:override_validation_key] = params[:validation_key] if params[:validation_key].present?
+
```

Comment:
Ensure override_validation_key is set when validation_key is present in edge cases.